app/api/report.py

```diff
@@ -1057,3 +1057,126 @@ def junior_monthly_report():
 
     return jsonify(report_all)
 
+
+#Api for get juniors managers details.
+@bp.route('/get_managers', methods=['GET'])

```

Comment:
move all 360 route to a file called 360.py
same for montly. all functions/routes for montly to montly.py